### DISCUSSION TITLE
Google Identity Services support

### DISCUSSION THREAD

**Discussion by webolot** [OpenIDC/mod_auth_openidc#850] [General]
Hi, today I received this email from Google...

> Hello Google Developer,
> 
> You are receiving this message because one or more of your web applications uses the legacy Google Sign-In web solution. Our announced plan earlier this month stated that authorization support for the Google Sign-In JavaScript platform library will no longer be supported after March 31, 2023.
> 
> Beginning April 30th, 2022, new applications must use the Google Identity Services library, while existing apps may continue using the Platform Library until the deprecation date.
> 
> After March 31, 2023, migration to the new Google Identity Services will be necessary to sign in customers using their Google Account to your platform. The SDK will use the new Sign In With Google client library.
> 
> What do I need to know?
> To help you find out which of your web apps or environments are affected, we’re sharing a list of your client ID(s) that use the legacy Google Sign-In web solution. This list should help you evaluate user impact and prioritize the migration work more efficiently:
> 
> App Name	Client ID
> OFFUSCATED	OFFUSCATED.apps.googleusercontent.com
> 
> What do I need to do?
> At your earliest convenience, migrate to the new Google Identity Services by following the migration guide.
> 
> Be sure to read through the Sign-In With Google resources guide. If you have migration specific feedback to share, send an email to gis-migration-feedback@google.com.
> 
> Thanks for choosing Google Identity Services.
> 
> –The Google Identity Services Team

I'm using mod_auth_openidc on apache. Here the config...

```
OIDCProviderMetadataURL https://accounts.google.com/.well-known/openid-configuration
OIDCClientID OFFUSCATED.apps.googleusercontent.com
OIDCClientSecret OFFUSCATED
OIDCRedirectURI https://www.OFFUSSCATED.com/OFFUSSCATED/
OIDCCryptoPassphrase OFFUSSCATED
OIDCScope "openid email profile https://www.googleapis.com/auth/admin.directory.group.member.readonly https://www.googleapis.com/auth/admin.directory.group.member https://www.googleapis.com/auth/admin.directory.group.readonly https://www.googleapis.com/auth/admin.directory.group"
OIDCSessionInactivityTimeout 900
```

Is there something to take care?

Is the mod_auth_openidc module ready to use Google Identity Services?

Thank's!

**Comment by zandbelt** [comment]
apparently you used a Javascript library for one of your apps that will no longer be supported, this does not have anything to do with mod_auth_openidc
